less/index.less

```diff
@@ -52,8 +52,345 @@ table {
+                            padding: 10px;
+                            border: 1.5px solid CornflowerBlue;
+                        }
+                        input {
+                            margin-bottom: 20px;
+                        }
+                    }
+                    .btn {
+                        padding: 5px 49px;
+                        margin: 16px 0px 0px;
+                        border: 1.5px solid #2e2e2e;
+                        background-color: #2e2e2e;
+                        color: #fff;
+                        font-size: 1.6rem;
+                        font-weight: bold;
+                    }
+                    .social {
+                        .column-left;
+                        // align-items: flex-start;
+                        ul {
+                            width: 100%;
+                            align-items: flex-start;
+                            li {
+                                padding: 10px 0px;
+                                a {
+                                    text-decoration: none;
+                                    font-size: 20px;
+                                    color: @clr-black;
+                                    .fab, .fa {
+                                        font-size: 24px;
+                                        padding-right: 15px;
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+
+            /* Footer */
+            footer {
+                background-color: rgba(22, 22, 22, .5);
+                margin-top: 30px;
+                text-align: center;
+                padding: 25px 0px 40px;
+                color: #fff;
+                font-size: .8rem;
+            }
+        }
+    }
+/* Mobile Query */
+@media (max-width: 650px) {
+    html, body {
+        .wrapper {
+            max-width: 100%;
+            .nav nav {
+                display: flex;
+                // align-items: center;
+                justify-content: space-between;
+                padding: 16px 10px;
+                position: fixed;
+                background-color: @clr-black;
+                width: 100%;
+                a {
+                    color: #fff;
+                    font-size: 1rem;
+                }
+                a:last-child {
+                    padding-right: 15px;
+                }
+            }
+            header {
+                padding: 0px 15px;
+                align-items: center;
+                text-align: center;
+                .sub-about {
+                    font-size: 1.2rem;
+                    padding-top: 20px;
+                }
+                .btn {
+                    margin-bottom: 30px;
+                    align-self: center;
+                }
+                img {
+                    margin-top: 30px;
+                    width: 100%;
```

Comment:
I noticed that your image is super stretched at your Mobile media queries. This can be fixed with `height: auto` or by having a mobile-specific image with the correct resolution/dimensions.